.ci/generate_bear_requirements.py

```diff
@@ -15,15 +15,26 @@
 
 import argparse
 import itertools
+import json
 import os
 import sys
 
 from pyprint.NullPrinter import NullPrinter
 
+from bears import VERSION
 from coalib.bears.BEAR_KIND import BEAR_KIND
 from coalib.collecting.Collectors import collect_bears
 
+from dependency_management.requirements.GemRequirement import GemRequirement
+from dependency_management.requirements.NpmRequirement import NpmRequirement
 from dependency_management.requirements.PipRequirement import PipRequirement
+from jinja2 import Environment, FileSystemLoader
```

Comment:
this import should appear together with pyprint , are they are both 'third party libraries'  (pyprint is owned by coala, but it isnt a tightly integrated package of coala)